### ISSUE TITLE
Premium package expiration and stantard package

### ISSUE THREAD

**Issue by davimlima** [Automattic/WP-Job-Manager#621]
I would like to know if it is possible to implement the following funcionality: when the employer's paid package (premium) expires, it turns to free package (stantard) automatically?

**Comment by mikejolley** [comment]
Not possible without customisation. The user would need to repost the expired listing with a 'free' package.

**Comment by davimlima** [comment]
I see...

Well, what about the comments and reviews attached to the prior listing?
The user loses it all?

I'm sorry, but I think no client would like to repost his listing... I had
to know it earlier... :/

Anyway, how can I customise it?

**Comment by mikejolley** [comment]
You can relist from the account page without losing data.

**Comment by davimlima** [comment]
I'm sorry, but I can't follow you on that one... What do you mean by relist
from the account page without losing data? How is that possible?

**Comment by mikejolley** [comment]
The job dashboard has a special 'relist' button. Click it :)

**Comment by davimlima** [comment]
I seee...

Sorry bro, I haven't got there yet. Since all my listings are free and
unlimited, I haven't seen this relist button... Awesome...

Sorry for the trouble...

Thanks...

**Comment by davimlima** [comment]
Hello again, Mike!

I just tested the relist button as you suggested and it seems it doesn't
charge to renew the selected package.

I created a package with 2 days duration for $2,00. I created a user and
purchased this package. Today it just expired and I clicked on relist, then
it was shown all packages avaiable and my current package was selected by
default.

I proceeded with the selected package as default and submitted my
listing... It was sent for approval without creating an order and charge
for it.

How can I fix it?

Thank you in advance.

**Comment by davimlima** [comment]
Hello again, Mike!

I just tested the relist button as you suggested and it seems it doesn't
charge to renew the selected package.

I created a package with 2 days duration for $2,00. I created a user and
purchased this package. Today it just expired and I clicked on relist, then
it was shown all packages avaiable and my current package was selected by
default.

I proceeded with the selected package as default and submitted my
listing... It was sent for approval without creating an order and charge
for it.

How can I fix it?

Thank you in advance.
